spec/unit/solr_doc_hash_spec.rb

```diff
@@ -1,420 +1,399 @@
-        :access_facet => 'Online',
-        :druid => druid,
-        :url_fulltext => "#{purl_url}/#{druid}",
-        :display_type => 'image',
-        :building_facet => 'Stanford Digital Repository'})
+    it 'returns an empty Array when there are no problems' do
+      hash = described_class.new(access_facet: 'Online',
+                                 druid: druid,
+                                 url_fulltext: "#{purl_url}/#{druid}",
+                                 display_type: 'image',
+                                 building_facet: 'Stanford Digital Repository')
       expect(hash.validate_gdor_fields(mock_config)).to eq([])
     end
-    it "should have a value for each missing field" do
-      hash = GDor::Indexer::SolrDocHash.new({})
+    it 'has a value for each missing field' do
+      hash = described_class.new({})
       expect(hash.validate_gdor_fields(mock_config).length).to eq(5)
     end
-    it "should have a value for an unrecognized display_type" do
-      hash = GDor::Indexer::SolrDocHash.new({
-        :access_facet => 'Online',
-        :druid => druid,
-        :url_fulltext => "#{purl_url}/#{druid}",
-        :display_type => 'zzzz', 
-        :building_facet => 'Stanford Digital Repository'})
+    it 'has a value for an unrecognized display_type' do
+      hash = described_class.new(access_facet: 'Online',
+                                 druid: druid,
+                                 url_fulltext: "#{purl_url}/#{druid}",
+                                 display_type: 'zzzz',
+                                 building_facet: 'Stanford Digital Repository')
       expect(hash.validate_gdor_fields(mock_config).first).to match(/display_type/)
     end
-    it "should have a value for access_facet other than 'Online'" do
-      hash = GDor::Indexer::SolrDocHash.new({
-        :access_facet => 'BAD',
-        :druid => druid,
-        :url_fulltext => "#{purl_url}/#{druid}",
-        :display_type => 'image', 
-        :building_facet => 'Stanford Digital Repository'})
+    it "has a value for access_facet other than 'Online'" do
+      hash = described_class.new(access_facet: 'BAD',
+                                 druid: druid,
+                                 url_fulltext: "#{purl_url}/#{druid}",
+                                 display_type: 'image',
+                                 building_facet: 'Stanford Digital Repository')
       expect(hash.validate_gdor_fields(mock_config).first).to match(/access_facet/)
     end
-    it "should have a value for building_facet other than 'Stanford Digital Repository'" do
-      hash = GDor::Indexer::SolrDocHash.new({
-        :access_facet => 'Online',
-        :druid => druid,
-        :url_fulltext => "#{purl_url}/#{druid}",
-        :display_type => 'image',
-        :building_facet => 'WRONG'})
+    it "has a value for building_facet other than 'Stanford Digital Repository'" do
+      hash = described_class.new(access_facet: 'Online',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.